dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -292,6 +292,7 @@
 
 # Allow emailconfirmed to skip captcha
 $wgGroupPermissions['emailconfirmed']['skipcaptcha'] = true;
+$wgAllowConfirmedEmail = true;
```

Comment:
So the docstring for this setting says it should still work, but is deprecated:

```php
/**
 * Allow users who have confirmed their email addresses to post
 * URL links without being harassed by the captcha.
 *
 * @deprecated since 1.36
 * $wgGroupPermissions['emailconfirmed']['skipcaptcha'] = true; should be used instead.
 */
$wgAllowConfirmedEmail = false;
```

While we set `$wgGroupPermissions['emailconfirmed']['skipcaptcha'] = true` above, I can confirm it's not working, but I think this is because its use is a bit more complicated than just adding the single line.

Looking here https://www.mediawiki.org/wiki/Manual:User_rights#Examples I think we need to do a bit of dancing to get it working.

```php
# Disable for everyone.
$wgGroupPermissions['*']['skipcaptcha'] = false;
# Disable for users, too: by default 'user' is allowed to edit, even if '*' is not.
$wgGroupPermissions['user']['skipcaptcha'] = false;
# Make it so users with confirmed email addresses are in the group.
$wgAutopromote['emailconfirmed'] = APCOND_EMAILCONFIRMED;
# Hide group from user list.
$wgImplicitGroups[] = 'emailconfirmed';
# Finally, set it to true for the desired group.
$wgGroupPermissions['emailconfirmed']['skipcaptcha'] = true;
```

I tested this locally and was able to add URLs as a random email confirmed user when doing the above.